syft/messaging/message.py

```diff
@@ -1,12 +1,14 @@
 import syft as sy
 
 from syft.workers import AbstractWorker
+from syft import codes
 
 
 class Message:
-    def __init__(self, msg_type, contents):
+    def __init__(self, msg_type, contents=None):
         self.msg_type = msg_type
-        self.contents = contents
+        if contents is not None:
+            self.contents = contents
```

Comment:
why do you want the member self.contents only to exist if contents is not None?

Reply:
mild efficiency - saves an op.

Reply:
(setter ops are less expensive than "is not" checking ops)

Reply:
But IMHO it is considered bad style to have classes with varying number of attributes.

Reply:
Ok - I think I came up with a good solution in https://github.com/OpenMined/PySyft/pull/2450/commits/d19f04bae919ac39753a6620b6658b739dc198b9